NHAIS/README.md

```diff
@@ -0,0 +1,45 @@
+# NHAIS
+
+NHAIS is a system that allows General Practice (GP) Surgeries to keep their patient registration and demographics data in sync with the regional Health Authorities (HA). Since the creation of this service the regional or area health authorities (approx 80) have since been replaced by a fewer number of successor organisations. There is however still a notion of every GP Practice’s patient being registered with one of the HAs.
+
+
+## Onboarding
+As outlined in section 2, the purpose of each adaptor is to enable access to national NHS Systems.  As an enabler, the adaptor will need to be configured to integrate with the relevant NHS environment(s).  Therefore you will need to register your intention to use adaptors through the standard NHS onboarding process which can be found here within the NHS Operations overview. 
+
+
+## Installation & Configuration inc dependencies 
+
+### Open Test and Path To Live Access
+
+In order to perform end-to-end testing of the MHS Adaptor, you will require access to an NHS Digital test environment. The MHS adaptor connects to the following two test endpoints which are available in an NHS Digital test environment:

```

Comment:
We mention MHS in this NHAIS document, is that ok?

Reply:
copy and paste special